css/css.css

```diff
@@ -71,3 +71,12 @@ div {
   -webkit-transform: scale(1.3) translateZ(0);
   transform: scale(1.3) translateZ(0);
 }
+
+.codeblock {
+  font-family: monospace
+  padding:2px 4px;
```

Comment:
Oops! I confess that I grabbed these right from the minified bootstrap file. Thanks for fixing it in your housekeeping commit.